src/js/components/Card.js

```diff
@@ -150,10 +216,12 @@ export default class Card extends Component {
 
 Card.propTypes = {
   thumbnail: PropTypes.string,
-  description: PropTypes.string,
-  heading: PropTypes.string,
+  textSize: PropTypes.oneOf(['small', 'medium', 'large', 'xlarge']),
```

Comment:
I felt `size` might be confusing. For the `Box` component, `size` adjusts the width of the box, while `textSize` adjusts the size of it's text content.